tests/pagila/copydb.sh

```diff
@@ -70,3 +79,6 @@ pgcopydb compare schema \
 pgcopydb compare data \
          --source ${PAGILA_SOURCE_PGURI} \
          --target ${PAGILA_TARGET_PGURI}
+
+psql -d ${PAGILA_TARGET_PGURI} -c "select last_value from identity_table_id_seq;" | grep -q 667
+psql -d ${PAGILA_TARGET_PGURI} -c "select last_value from normal_table_id_seq;" | grep -q 667
```

Comment:
This will be easier to implement in the tests/unit parts, you'll see. We have a notion of expected output to compare to using diff, etc.

Reply:
Yeah, it is a little easier :-). I'll try to use that more often.